src/components/AbstractWidget.js

```diff
@@ -26,6 +26,13 @@ export default class AbstractWidget extends React.Component {
     const $script = require('scriptjs')
 
     $script.ready('twitter-widgets', () => {
+      if (!window.twitter) {
```

Comment:
Oh god that's embarrassing. I wrote and tested this while on the plane (where my goal was to avoid the JS errors) and verified that worked. Didn't check again when I got off the plane. :P

Anyways, fixed!